src/demo/demo.test.js

```diff
@@ -0,0 +1,46 @@
+import { h } from 'preact'
+import { mount } from 'enzyme'
+import { expect } from 'chai'
+
+describe('Mount Demo App by mocking Onfido SDK', () => {
+  let Demo = null
+
+  beforeEach(() => (Demo = require('./demo.js')))
```

Comment:
We have duplicated `beforeEach()` here, which can be clean-up by grouping the current 2 `describe()`'s under a higher-level `describe()` and place this `beforeEach()` into:
```suggestion
  describe('Demo app', () => {
    beforeEach(() => {
      Demo = require('./demo.js')
    })
    
    describe('Mount Demo App by mocking Onfido SDK', () => {
      ...
    })
    
    describe('Mount Demo App without mocking Onfido SDK', () => {
      beforeEach(() => {
        // when the Onfido SDK is imported Woopra needs to be mocked
        const mockWoopraFn = jest.fn()
        ...
      })
      
      ...
    })
  })
```